Source/WebCore/crypto/keys/CryptoKeyHMAC.cpp

```diff
@@ -126,15 +126,14 @@ JsonWebKey CryptoKeyHMAC::exportJwk() const
     return result;
 }
 
-ExceptionOr<size_t> CryptoKeyHMAC::getKeyLength(const CryptoAlgorithmParameters& parameters)
+ExceptionOr<std::optional<size_t>> CryptoKeyHMAC::getKeyLength(const CryptoAlgorithmParameters& parameters)
 {
     auto& aesParameters = downcast<CryptoAlgorithmHmacKeyParams>(parameters);
 
-    size_t result = aesParameters.length ? *(aesParameters.length) : getKeyLengthFromHash(aesParameters.hashIdentifier);
-    if (result)
-        return result;
-
-    return Exception { ExceptionCode::TypeError };
+    std::optional<size_t> result = aesParameters.length ? *(aesParameters.length) : getKeyLengthFromHash(aesParameters.hashIdentifier);
```

Comment:
Would be nice to rename `result` to `length` to match the code above since we're making a change here.

Reply:
Also could use `auto` instead of `std::optional<size_t>`.